database/migrations/1641067337881_add_instagram_ref_and_cell_phone_number_columns.ts

```diff
@@ -0,0 +1,19 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
+export default class Users extends BaseSchema {
+  protected tableName = 'users'
+
+  public async up() {
+    this.schema.alterTable(this.tableName, (table) => {
+      table.string('instagram_ref')
+      table.string('cell_phone_number').notNullable()
```

Comment:
```suggestion
      table.string('cell_phone_number')
```
remove this thing, the `users` table is aready in production...